linera-core/src/worker.rs

```diff
@@ -413,12 +413,12 @@ where
     ) -> Result<CrossChainRequest, WorkerError> {
         let heights =
             BTreeSet::from_iter(height_map.iter().flat_map(|(_, heights)| heights).copied());
-        let mut keys = Vec::new();
+        let mut heights_red = Vec::new();
```

Comment:
What does the `red` stand for?
Maybe `heights_usize` would be better?

Reply:
Ok.